README.md

```diff
@@ -21,51 +21,52 @@ is now under the drand organization.
 audit yet. Therefore, DO NOT USE it in production or for anything security
 critical at this point.**
 
-# Table of Contents
-
-   * [Drand - A Distributed Randomness Beacon Daemon](#drand---a-distributed-randomness-beacon-daemon)
-      * [Disclaimer](#disclaimer)
-      * [Table of Contents](#table-of-contents)
-      * [Goal and Overview](#goal-and-overview)
-         * [Public Randomness](#public-randomness)
-         * [Private Randomness](#private-randomness)
-      * [Local demo](#local-demo)
-      * [Installation](#installation)
-         * [Official release](#official-release)
-         * [Manual installation](#manual-installation)
-            * [Via Golang](#via-golang)
-            * [Via Docker](#via-docker)
-         * [TLS setup: Nginx with Let's Encrypt](#tls-setup-nginx-with-lets-encrypt)
-      * [Usage](#usage)
-         * [Setup](#setup)
-            * [Long-Term Key](#long-term-key)
-            * [Starting drand daemon](#starting-drand-daemon)
-               * [With TLS](#with-tls)
-               * [Without TLS](#without-tls)
-            * [Run the setup phase](#run-the-setup-phase)
-         * [Distributed Key Generation](#distributed-key-generation)
-         * [Randomness Generation](#randomness-generation)
-         * [Control Functionalities](#control-functionalities)
-            * [Long-Term Private Key](#long-term-private-key)
-            * [Long-Term Public Key](#long-term-public-key)
-            * [Private Key Share](#private-key-share)
-            * [Distributed Key](#distributed-key)
-         * [Using Drand](#using-drand)
-         * [Fetching Public Randomness](#fetching-public-randomness)
-            * [Fetching Private Randomness](#fetching-private-randomness)
-            * [Using HTTP endpoints](#using-http-endpoints)
-         * [Updating Drand Group](#updating-drand-group)
-      * [Metrics](#metrics)
-      * [DrandJS](#drandjs)
-      * [Documentation](#documentation)
-      * [What's Next?](#whats-next)
-      * [License](#license)
-      * [Contributors](#contributors)
-      * [Acknowledgments](#acknowledgments)
-      * [Coverage](#coverage)
-      * [Supporting](#supporting)
-
-
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
is the comment auto-generated by doctoc as well? can we have that automated somehow? (e.g. as a linting process that runs it in ci and makes sure it's kept up-to-date)

Reply:
I'm not sure how to make it auto generated, doctoc seems to use pre-commit, which is not a normal thing. Also on circle ci, it would be a check, but can it modifies files as well and commit that on the branch ? 
If somebody knows how to do it fast, then yes sure, but I would prefer to avoid spending time *now* searching for this... :pray: